recipes/expat/all/test_package/conanfile.py

```diff
@@ -1,17 +1,25 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, cmake_layout
 import os
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "cmake_find_package"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
```

Comment:
```suggestion
    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
```

For my own education, I don't see any explicit call to scripts generated by `VirtualRunEnv`, where are they implicitly called in this conanfile?

Reply:
For CMake, here is the deal https://docs.conan.io/en/2.0/reference/tools/cmake/cmake.html
For VirtualRunEnv, here you see the file names that it generates: https://docs.conan.io/en/2.0/reference/tools/env/virtualrunenv.html and here you can see the `run()` part https://docs.conan.io/en/2.0/reference/tools/env/envvars.html#running-with-environment-files

I agree we have to document it better and crosslink both sections. Thanks for the feedback.